### DISCUSSION TITLE
React-final-form submitting is not reset

### DISCUSSION THREAD

**Discussion by NLSVTN** [broadinstitute/seqr#3211] [Q&A] [answered]
Hi,

We are having an identical issue to the following issue that is still open on react-final-form github:

https://github.com/final-form/react-final-form/issues/903

If we go to variant search page, do a search, then modify parameters and try to resubmit the form - it is not working. I debugged react-final-form code and the form does not reset **formState.submitting** parameter. I tested both **6.5.8** and **6.5.9** versions - neither of them is working. Have you faced the issue? If resubmitting is working fine for you, do you know what could be causing **formState.submitting** not being reset?

**Comment by hanars** [comment, marked as answer]
We do not have this issue in the main seqr instance. Because we don't have this issue, I unfortunately don't know why you might be, although the one idea I can think of is it might have to do with error handling- if there are no errors the state is properly managed but but maybe if there are unhandled errors the state is not setting correctly. I recommend updating to the latest version of seqr and keeping as up to date as possible, and seeing if that fixes your issue
